apis/v1alpha2/types.go

```diff
@@ -71,7 +71,7 @@ type ResourceGroupStatus struct {
 	// TODO(negz): Do we really need the name in both spec and status?
```

Comment:
Looks like you are solving this by adding `omitempty`, okay to remove `TODO`?

Reply:
I think the omitempty is something @soorena776 added due to issues serialising statuses? I'd prefer to leave the TODO as I think it's still relevant.

Reply:
Sounds reasonable to me!